examples/react/react-native-app/android/app/build.gradle

```diff
@@ -79,7 +79,7 @@ import com.android.build.OutputFile
  */
 
 project.ext.react = [
-    entryFile: "examples/react/react-native-app/src/main.tsx",
```

Comment:
are you running the nx commands from the root or from the example app's directory? that could explain why it wouldn't be working

Reply:
ah I was running it from the example apps directory, not root. so nvm will close this!
(I had tested by regenerating a new RN app thru nx and it has the entry file set to `src/main.tsx`)